From 3bd8add6a0661edd1ce1a2a818a976f283cee2d7 Mon Sep 17 00:00:00 2001
From: Stefan Wahren <wahrenst@gmx.net>
Date: Thu, 19 Sep 2019 20:45:30 +0200
Subject: [PATCH 0180/1085] ARM: bcm: Switch board, clk and pinctrl to bcm2711
 compatible

After the decision to use bcm2711 compatible for upstream, we should
switch all accepted compatibles to bcm2711. So we can boot with
one DTB the down- and the upstream kernel.

Signed-off-by: Stefan Wahren <wahrenst@gmx.net>
---
 arch/arm/mach-bcm/board_bcm2835.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/mach-bcm/board_bcm2835.c
+++ b/arch/arm/mach-bcm/board_bcm2835.c
@@ -109,7 +109,7 @@ static const char * const bcm2835_compat
 #ifdef CONFIG_ARCH_MULTI_V7
 	"brcm,bcm2836",
 	"brcm,bcm2837",
-	"brcm,bcm2838",
+	"brcm,bcm2711",
 #endif
 	NULL
 };
